themes/src/main/resources/theme/base/login/template.ftl

```diff
@@ -151,4 +151,20 @@
   </div>
 </body>
 </html>
+
+<script>
+    function preventDoubleClick(){
+        const submitButtons = document.querySelectorAll('input[type="submit"]');
+        for (var i = 0; i < submitButtons.length; i++) {
+            const submitButton = submitButtons[i];
+            submitButton.disabled = true;
+        }
+        return true;
+    }
+    const forms = document.getElementsByTagName('form');
+    for (var i = 0; i < forms.length; i++) {
+        const form = forms[i];
+        form.addEventListener("submit", preventDoubleClick)
+    }
```

Comment:
Since [`HTMLCollection`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLCollection) is iterable we can use a [`for...of`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of) loop here instead.

```suggestion
    for (const form of forms) {
        form.addEventListener('submit', preventDoubleClick);
    }
```

Reply:
Done